pennylane/qaoa/cycle.py

```diff
@@ -17,13 +17,16 @@
 # pylint: disable=unnecessary-comprehension
 import itertools
 from typing import Dict, Tuple, Iterable, List
+
 import networkx as nx
+import retworkx as rx
+
 import numpy as np
 import pennylane as qml
 from pennylane.ops import Hamiltonian
 
 
-def edges_to_wires(graph: nx.Graph) -> Dict[Tuple, int]:
+def edges_to_wires(graph) -> Dict[Tuple, int]:
```

Comment:
Assuming we have `from typing import Union`
we can do
```suggestion
def edges_to_wires(graph: Union[nx.Graph, rx.Graph]) -> Dict[Tuple, int]:
```

Reply:
Ah! Thanks a lot @mlxd for catching this. I fixed this in all places.